### DISCUSSION TITLE
Version Openaudible 3.6.2. to 3.7.2

### DISCUSSION THREAD

**Discussion by eva-roxi** [openaudible/openaudible#1175] [General]
In version 3.6.2 the conversion from AAX to MP3 works with the directories: Library, Web Directory and Preferences.
In version 3.7.2 the directory Library is missing.
Therefore the conversion to MP3 does not work because the required directory is not found.

**Comment by openaudible** [comment]
3.7.2 uses a File: Manage Libraries feature where you can set, change, or move library directories.
